src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/CachePoolPass.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\DefinitionDecorator;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+class CachePoolPass implements CompilerPassInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        $attributes = array(
+            'provider',
+            'namespace',
+            'default_lifetime',
+        );
+        foreach ($container->findTaggedServiceIds('cache.pool') as $id => $tags) {
+            $adapter = $pool = $container->getDefinition($id);
+            if ($pool->isAbstract()) {
+                continue;
+            }
+            if (!isset($tags[0]['namespace'])) {
```

Comment:
Could it make sense to apply several ``cache.pool`` tags on the service ? If yes, we need to handle all tags, not only the first one.
If not, we may want to warn the user that subsequent tags are ignored to ease debugging